crates/typos-dict/assets/words.csv

```diff
@@ -49,11 +49,11 @@ abberivates,abbreviates
 abberivation,abbreviation
 abberration,aberration
 abberviation,abbreviation
-abbort,abbot,abort
+abbort,abort,abbot
```

Comment:
I did a scan through this.  I might have missed something but it looks like no edits were reverted because of this change.  The use of allowed words helps.